src/components/layout/Toolbar.vue

```diff
@@ -1,4 +1,10 @@
 <template>
+  <div v-if="(activeRole && activeRole != 'default') || (!activeRole && roles.length > 0)">
+    <Message severity="warn">
+      When making API requests, you are currently acting as a user with the [{{ activeRole || roles.join(", ") }}] role(s). You may have additional permissions than normal. To change your
```

Comment:
should be `You may have additional permissions` or `You may have more permissions than normal` but not both :P